table/tables/tables.go

```diff
@@ -584,12 +575,14 @@ func (t *Table) removeRowIndices(ctx context.Context, h int64, rec []types.Datum
 		vals, err := v.FetchValues(rec)
 		if vals == nil {
 			// TODO: check this
```

Comment:
Is this a stale TODO ?

Reply:
Nop, If the index is public, I think we need handle it.